src/bin/index.js

```diff
@@ -1,21 +1,40 @@
 #!/usr/bin/env node
-
+import * as path from 'path';
+import { promises as fs } from 'fs';
 import { MarkdownDB } from "../lib/markdowndb.js";
 
+async function loadConfig(configFilePath) {
+  const normalizedPath = path.resolve(configFilePath);
+  const fileUrl = new URL(`file://${normalizedPath}`);
+
+  try {
+    // Import the module using the file URL
+    const configModule = await import(fileUrl.href);
+    return configModule.default;
+  } catch (error) {
+    throw new Error(`Error loading configuration file from ${normalizedPath}: ${error.message}`);
+  }
+}
+
 // TODO get these from markdowndb.config.js or something
 const dbPath = "markdown.db";
 const ignorePatterns = [/Excalidraw/, /\.obsidian/, /DS_Store/];
-const [contentPath] = process.argv.slice(2);
+const [contentPath, configFilePath] = process.argv.slice(2);
```

Comment:
What happens if you don't have a config file - this stuff should work with defaults ...